source/_components/keba.markdown

```diff
@@ -0,0 +1,97 @@
+---
+layout: page
+title: "Keba Charging Station"
+description: "Instructions on how to setup your Keba charging station with Home Assistant."
+date: 2019-06-11 20:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: keba.png
+ha_category:
+  - Binary Sensor
+  - Lock
+  - Sensor
+ha_release: 0.95
+---
+
+The `keba` integrates your Keba charging station (wallbox) into your home assistant instance. It was tested with a BMW Wallbox but should also work with a Keba P20/P30 according to the developers [manual](https://www.keba.com/web/downloads/e-mobility/KeContact_P20_P30_UDP_ProgrGuide_en.pdf){:target="_blank"}. The fetching interval to the charging station is set to 5 seconds, same as in the official mobile app.
+
+This component provides the following platforms:
+
+- Binary Sensors: Online state, plug state, Charging state and failsafe mode state.
+- Lock: Authorization (like with the RFID card).
+- Sensors: current set by the user, target energy set by the user, charging power, charged energy of the current session and total energy charged.
+- Services: authorize, deauthorize, set energy target, set maximum allowed current and manually update the states. More details can be found [here](/components/keba/#services).
+
+## {% linkable_title Configuration %}
+
+To enable this component in your installation, add the following to your
+`configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+keba:
+  host: KEBA_HOST
+```
+
+{% configuration %}
+keba:
+  description: configuration
+  required: true
+  type: map
+  keys:
+    host:
+      description: Keba host.
+      required: true
+      type: string
+    rfid:
+      description: RFID tag used for authorization.
+      required: false
+      type: string
+      default: "00845500"
+    failsafe:
+      description: Enable failsafe mode at home assistant startup.
+      required: false
+      type: boolean
+      default: false
+    failsafe_timeout:
+      description: Timeout of the failsafe mode in seconds. Allowed values are between 10 seconds and 600 seconds (only if failsafe mode is enabled). Make sure to call the `keba.set_curr` service regularly within this timeout period!
+      required: false
+      type: integer
+      default: 30
+    failsafe_fallback:
+      description: Fallback current of the failsafe mode in A. Allowed values are between 6 Ampere and 63 Ampere. 0 Ampere disables the running charging process completely (only if failsafe mode is enabled).
+      required: false
+      type: integer
+      default: 6
+    failsafe_save:
+      description: Saving the failsafe configuration to internal EEPROM of the Keba charging station. 1 means save it, 0 means do only keep this configuration until the next restart of the charging station (only if failsafe mode is enabled).
+      required: false
+      type: integer
+      default: 0
+{% endconfiguration %}
+
+## {% linkable_title Services %}
```

Comment:
Maybe put the service name in the title

Reply:
I am not 100% sure what your suggestion is, but I now added the service names in the descritpion of the service and additionally added some example payloads to the services calls with parameters.